cmd/tsconnect/src/lib/ssh.ts

```diff
@@ -27,26 +27,39 @@ export function runSSHSession(
 
   term.focus()
 
-  const sshSession = ipn.ssh(def.hostname, def.username, {
-    writeFn: (input) => term.write(input),
-    setReadFn: (hook) => (onDataHook = hook),
+  let resizeObserver: ResizeObserver | undefined
```

Comment:
It was necessary to add declarations for these earlier than their use below because `onDone` can be invoked synchronously, before they're initialized.